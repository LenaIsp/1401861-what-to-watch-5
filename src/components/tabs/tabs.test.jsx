import React from 'react';
import renderer from 'react-test-renderer';
import {TEST_MOCKS, TEST_MOCK_COMMENT, TEST_MOCK_MOVIE} from "../../test-mock.js";
import Tabs from './tabs';

it(`ShowMore is rendered correctly`, () => {
  const tree = renderer.create(
      <Tabs
        rating={TEST_MOCK_COMMENT.rating}
        director={TEST_MOCK_MOVIE.director}
        scoresCount={TEST_MOCK_MOVIE.scoresCount}
        description={TEST_MOCK_MOVIE.description}
        starring={TEST_MOCK_MOVIE.starring}
        released= {TEST_MOCK_MOVIE.released}
        runTime={TEST_MOCK_MOVIE.runTime}
        handleClick={TEST_MOCKS.noop}
        reviews={TEST_MOCKS.reviews}
        genre={TEST_MOCKS.genreFilter}
        state={TEST_MOCKS.state}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
