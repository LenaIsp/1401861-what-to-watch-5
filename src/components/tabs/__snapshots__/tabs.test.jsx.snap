// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShowMore is rendered correctly 1`] = `
<div
  className="movie-card__desc"
>
  <nav
    className="movie-nav movie-card__nav"
  >
    <ul
      className="movie-nav__list"
    >
      <li
        className="movie-nav__item movie-nav__item--active"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          Overview
        </a>
      </li>
      <li
        className="movie-nav__item"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          Details
        </a>
      </li>
      <li
        className="movie-nav__item"
        onClick={[Function]}
      >
        <a
          className="movie-nav__link"
          href="#"
        >
          Reviews
        </a>
      </li>
    </ul>
  </nav>
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    >
      3.3
    </div>
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Normal
      </span>
      <span
        className="movie-rating__count"
      >
        1635992
         ratings
      </span>
    </p>
  </div>
  <div
    className="movie-card__text"
  >
    <p>
      A young man who was sentenced to seven years in prison for robbing a post office ends up spending three decades in solitary confinement. During this time, his own personality is supplanted by his alter-ego, Charles Bronson.
    </p>
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
        Nicolas Winding Refn
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
         Tom Hardy,
         Kelly Adams,
         Luing Andrews and other
      </strong>
    </p>
  </div>
</div>
`;
